modules/kubeconfig/main.tf

```diff
@@ -19,11 +19,24 @@ data "template_file" "kubeconfig_provider" {
   }
 }
 
+# 1.24 change: Create service account secret
+resource "kubernetes_secret" "sa_secret" {
+  metadata {
+    name      = local.service_account_secret_name
+    namespace = var.namespace
+    annotations = {
+      "kubernetes.io/service-account.name" = local.service_account_name
+    }
+  }
+
+  type = "kubernetes.io/service-account-token"
+}
+
 # Service Account based kube config data/template/resources
```

Comment:
Is this comment still true?

Reply:
This comment is true because the Service Account is still based off the template. The secret for the service account is the new resource created.